### PR TITLE
First version

### PR DESCRIPTION
**Pull request by yuqinggongyg**
I create index.html, please review, thank you.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div>
+    <h1>hello it is me</h1>
```
Comment (MoxHe) [inline index.html:2]: nice

Comment (MoxHe) [reply index.html:2]: good job!

**Review by MoxHe — COMMENTED** [review]
looks good overall. just a couple small comments. Please addrese them

**Review by MoxHe — CHANGES_REQUESTED** [review]
Please change

**Comment by yuqinggongyg** [comment]
> Please change



> Please change

Updated entire project codes.

File: index.html (hunk 1 above)
Comment (yuqinggongyg) [reply index.html:2]: yes it is from Adele!

**Review by MoxHe — APPROVED** [review]
LGTM!

**Pull request merged**